app/data/mounts.json

```diff
@@ -225,6 +225,13 @@
             "spellid": 342335,
             "icon": "inv_horse2bastionmount_blue",
             "itemId": 183741
+          },
+          {
+            "ID": 1414,
+            "name": "Sinrunner Blanchy",
```

Comment:
Why in treasures? This is a rare spawn according to wowhead.

Reply:
Is now a rare spawn.